java-chassis-dependencies/default/pom.xml

```diff
@@ -31,70 +31,94 @@
+    <javax-resource.version>1.5</javax-resource.version>
+    <javax-servlet.version>4.0.1</javax-servlet.version>
+    <javax-transaction.version>1.2</javax-transaction.version>
+    <javax-validation.version>2.0.1.Final</javax-validation.version>
+    <javax-ws-rs.version>2.0.1</javax-ws-rs.version>
+    <jedis.version>2.9.0</jedis.version>
+    <jersey.version>2.22.2</jersey.version>
+    <jettison.version>1.3.8</jettison.version>
+    <jmockit.version>1.30</jmockit.version>
+    <json-lib.version>2.4</json-lib.version>
+    <junit.version>4.12</junit.version>
     <log4j.version>1.2.17</log4j.version>
     <log4j2.version>2.11.2</log4j2.version>
-    <commons.io.version>2.6</commons.io.version>
-    <javassist.version>3.24.0-GA</javassist.version>
-    <javax.ws.rs.version>2.0.1</javax.ws.rs.version>
-    <javax.servlet.version>4.0.1</javax.servlet.version>
-    <apache.httpclient>4.5.7</apache.httpclient>
+    <logback.version>1.2.3</logback.version>
+    <mockito.version>1.10.19</mockito.version>
+    <mock-server.version>3.10.4</mock-server.version>
+    <narayana.version>5.3.2.Final</narayana.version>
+    <netflix-commons.version>0.3.0</netflix-commons.version>
+    <netty.version>4.1.28.Final</netty.version>
+    <okhttp3.version>3.11.0</okhttp3.version>
+    <powermock.version>1.6.2</powermock.version>
+    <protobuf.version>3.6.1</protobuf.version>
     <protostuff.version>1.5.9</protostuff.version>
+    <protostuff-parser.version>2.2.25</protostuff-parser.version>
+    <ribbon.version>2.2.5</ribbon.version>
+    <rxjava.version>1.1.6</rxjava.version>
+    <rxnetty.version>0.4.9</rxnetty.version>
+    <seanyinx.version>1.0.0</seanyinx.version>
+    <servo.version>0.12.25</servo.version>
+    <simpleclient.version>0.3.0</simpleclient.version>
+    <simpleclient-httpserver.version>0.1.0</simpleclient-httpserver.version>
+    <slf4j.version>1.7.26</slf4j.version>
+    <snakeyaml.version>1.23</snakeyaml.version>
+    <spectator.version>0.83.0</spectator.version>
+    <spring.version>4.3.20.RELEASE</spring.version>
+    <spring-boot.version>1.5.12.RELEASE</spring-boot.version>
+    <spring-cloud.version>Edgware.SR3</spring-cloud.version>     <!-- Spring Zuul handling of gzip response has bug in version Dalston.SR1 -->
     <swagger.version>1.5.22</swagger.version>
-    <netty.version>4.1.28.Final</netty.version>
+    <swagger2markup.version>1.3.3</swagger2markup.version>
     <tcnetty.version>2.0.14.Final</tcnetty.version>
-    <main.basedir>${basedir}/../..</main.basedir>
-    <narayana.version>5.3.2.Final</narayana.version>
-    <cxf.version>3.1.6</cxf.version>
-    <logback.version>1.2.3</logback.version>
-    <brave.version>5.6.0</brave.version>
+    <tec-zkclient.version>0.10</tec-zkclient.version>
+    <tomakehurst.version>2.6.0</tomakehurst.version>
+    <vertx.version>3.6.3</vertx.version>
+    <woodstox.version>4.4.1</woodstox.version>
+    <xstream.version>1.4.9</xstream.version>
```

Comment:
xstream should use 1.4.11.1

Reply:
Just in case, this PR is just for setting versions as properties, all versions are the same as previous. Updating version will be done after this PR.

Reply:
Hi, do you mean this pull request is just for the dependencies rearrangement, and the dependencies upgradation is in your next pull request?

Reply:
Yes, that is exactly what I meant.